src/Features/Onboarding.php

```diff
@@ -112,7 +112,7 @@ public static function get_allowed_industries() {
 			array(
 				'fashion-apparel-accessories' => __( 'Fashion, apparel & accessories', 'woocommerce-admin' ),
 				'health-beauty'               => __( 'Health & beauty', 'woocommerce-admin' ),
-				'art-music-photography'       => __( 'Art, music, & photography', 'woocommerce-admin' ),
+				'art-music-photography'       => __( 'Art, music & photography', 'woocommerce-admin' ),
```

Comment:
Nitpicky, but personally, I'm a fan of the Oxford comma to reduce ambiguity.  This sounds a bit like `music & photography` are grouped together.

Is there a thread on this somewhere or precedent for this style?

Reply:
I personally prefer the oxford comma as well. This request came in through p90Yrv-1iA-p2 so that it would match `Fashion, apparel & accessories` and `Home, furniture & garden`

However, I just searched the Field Guide and both our WP.com and A8C style guides say we should use the oxford comma -- so I'll go ahead and update these before merge.